src/CustomerProfile.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from "react";
+
+const CustomerProfile = props => {
+  const [profile, setProfile] = useState("");
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/customers/${props.id}`)
+      .then(res => res.json())
+      .then(data => {
+        setProfile(data);
```

Comment:
Good job setting the state for this component and correctly getting `useEffect` to run again if the selected customer id changes.